ports/zephyr/machine_timer.c

```diff
@@ -0,0 +1,196 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * Development of the code in this file was sponsored by Microbric Pty Ltd
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2013-2015 Damien P. George
+ * Copyright (c) 2016 Paul Sokolovsky
+ * Copyright (c) 2025 Daniel Campora on behalf of REMOTE TECH LTD
```

Comment:
As with #16793, I suggest to only copy across copyright lines where you took "substantial portions of the Software" from somewhere else.

Reply:
@done.